static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -12,48 +12,66 @@ import commercialFeatures from 'commercial/modules/commercial-features';
 import IncreaseInlineAds
     from 'common/modules/experiments/tests/increase-inline-ads';
 
+type AdSize = {
```

Comment:
With flow, yes, you can export and import types.
With the linter, the export type and import type keywods make it blow up in the worst possible way (with no actual proper error message).
There might be another way to get the types flowing through without exporting or importing.